common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -159,10 +159,16 @@ def verify_profile_forced_private_message(self, username, birth_year, message=No
         """
         Verify age limit messages for a user.
         """
-        self.set_birth_year(birth_year=birth_year if birth_year is not None else "")
+        if birth_year:
```

Comment:
I'm a fan of the ``foo if test else bar`` idiom, as it is concise and makes it clearer that it is just the one value that is different. Separating out two clauses means you have to read both carefully to be certain that they are actually doing the same thing.

Also, we try to avoid using falsiness for numbers, because 0 is falsey and we've had several situations where it was treated incorrectly. I suppose this isn't too likely for birth year, but for consistency I'd stick with ``birth_year is not None``.